### ISSUE TITLE
Setting initial value for attribute AND performing ADD operation within single `update` call on an Entity object

### ISSUE THREAD

**Issue by aorumbayev** [tywalch/electrodb#297]
Suppose there is the following update call in dynamo db client:

```
table.update_item(
    Key={
        'key': my_key
    },
    UpdateExpression="SET my_value = if_not_exists(my_value, :initial_value) + :increment",

    ExpressionAttributeValues={
        ':inc': initial_value,
        ':increment': 0,
    },
    ReturnValues="UPDATED_NEW"
)
```

If i were to translate that to ElectroDB syntax, it would not be possible to have both:

```
SomeEntity.update({ userID, assetID })
      .data((attr, op) => {
        op.ifNotExists(attr.my_value, initial_value);
        op.Add(attr.my_value, increment);
      }).go()
```

Within the same statement, during execution it will generate errors since attr.my_value will be referenced in UpdateExpression twice. The only way to resolve this at the moment seems to be to overwrite the UpdateStatement completely using the `params` object inside `go` method. But that seems rather hacky way to deal with that. 

Hence my main question is, is there any other way to use electroDB syntax that will result in a desired UpdateExpression statement that sets initial value (if not exists) and then perform add?

### Version of electrodb used

`2.9.3`

**Comment by tywalch** [comment]
Hi @aorumbayev 👋 

Thanks for reaching out! This is actually going to be included in the next update!

**Comment by aorumbayev** [comment]
@tywalch amazing! Thanks for quick reply! What's the best way to track when that release is out?

**Comment by tywalch** [comment]
I just linked this issue to the PR that it will get wrapped into

**Comment by tywalch** [comment]
Merged this with `2.10.0`, here is a [playground](https://electrodb.fun/?#code/PQKgBAsg9gJgpgGzARwK5wE4Es4GcA0YuccYGeqCALgUQBYCG5YA7llXWAGbZwB2MXGBDAAUKKwBbAA5QMVMAG8wAUT5V2AT0IBlTADcsAY1IBfbhiiSwAIkRwjVSzABGNgNzijUPrgVUGFwRSAF5bTShUDAB9AKC4aL4GSTgPcVAwABUGXABrITUNKk1hMW9ffxz8sDC+OBZVdS0AClEwJTb2sElYRAAuDq6u-iLNAZsAvNwbfE6h-UxcLB9xgEYZua7iDEMTccnchmlpG03TWaGGKicsF1QqPAHFTfaH5KeXruLpOHG-bD4AHMNkMhuQ0FhyDABk50J9zp8Dh9QV9ND8-jcgSCUWQ4BCoTCMHCUQiUdJLAArBxUZE476-Wz-LBYi448GoSFwaFgWFweGsoaoba0lH0jEA4EC0Hsznc3n8xHsYIi0FixmYyWfdoygk8olwKXtUmg+C4IzYaQaFaDOlohk2JlYhUovxXIUqobAYBgIWkBh8MBMDAMEpUKA8u2BgP8VCSLUR9FgADaNigPz4M1szIAtOSoIDyLhpoRU3xs3QoAgYJmbEYEFBiNWALqBoTlPyGrrwLgMSg02xp-inEmd2TMmge1GJmx8WMuTDYoamM5S5nwAAejxtQzzVMcuEn7WkuUPXS4OCr42Pi5x3hkDfYDJTb0kNib8eNLpP25x58Q3IdXIbxRL0wHNOArlIO9ZCWB4wFyOBNCELg5DAaQmA0BgkFwOQFAQko0EwHBcHjdpoIfB4BhTXdqRrA43w-eEzk6Y1lDiYIwGXABKTxRHbfwIOsMIbALEgM08fi0MpakalsbxyDSSSDlkmwABYAFZswANi0gAmAAGNJRAOXAADp0KoIw6GaRQaMcQgDgcwTTC4zpTJgK4GGaGy0KgcchHOJRAxgGBOK4moAD5gpgZox3UWhVMIdSuNc9pTMBKBmh4oA) showing it's use -- `add` and `subtract` now accept a third parameter allowing you to specify a `defaultValue`. Let me know if you have any issues!
